src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.tsx

```diff
@@ -70,7 +71,19 @@ export const ProposalActionsAction: React.FC<IProposalActionsActionProps> = (pro
                     <ProposalActionsActionVerification action={action} />
                 </div>
             </Accordion.ItemHeader>
-            <Accordion.ItemContent>{ActionComponent}</Accordion.ItemContent>
+            <Accordion.ItemContent>
+                {action.value != null && action.value !== '0' && (
```

Comment:
`action.value` cannot be null as from the interface definition. Also we shouldn't display the alert when action has no data (meaning that the action is a native transfer)